app/components/card-hazard.tsx

```diff
@@ -19,31 +42,58 @@ const CardHazard: React.FC<CardHazardProps> = ({ hazard, hazardData }) => {
   const { title, name, description } = hazard;
   const { exists, last_updated: date } = hazardData || {};
 
-  const cardProps = {
-    header: (
-      <HStack>
-        <Text textStyle="textBig">{title}</Text>
-      </HStack>
-    ),
-    footer: (
-      <HStack justifyContent="space-between" width="100%">
-        <Text
-          cursor="pointer"
-          textStyle="textMedium"
-          color="lightBlue"
-          textDecoration="underline"
-        >
-          More info
-        </Text>
-        {exists !== undefined ? <Pill exists={exists} /> : ""}
-      </HStack>
-    ),
+  const buildHazardCardInfo = () => {
+    return (
+      <VStack gap={5} p={5}>
+        {hazard.info.map((infoItem, index) => (
+          <Text key={index}>{infoItem}</Text>
+        ))}
+      </VStack>
+    );
   };
 
   return (
-    <BaseCard {...cardProps}>
-      <Text textStyle="textMedium">{description}</Text>
-    </BaseCard>
+    <Card flex={1} maxW={400} p={{ base: "16px", md: "20px" }}>
+      <Popover placement="bottom" returnFocusOnClose={false} closeOnBlur={true}>
+        <PopoverTrigger>
+          <VStack cursor={"pointer"} alignItems={"flex-start"} h={"100%"}>
+            <CardHeader p={0}>
+              <Text textStyle="textBig" fontWeight={"700"}>
+                {title}
+              </Text>
+            </CardHeader>
+            <CardBody p={0} mb={"14px"}>
+              <Text>{description}</Text>
+            </CardBody>
+            <CardFooter p={0} width={"100%"}>
+              <HStack justifyContent="space-between" width="100%">
+                <Text cursor={"pointer"} textDecoration={"underline"}>
+                  More Info
+                </Text>
+                {exists !== undefined ? <Pill exists={exists} /> : ""}
+              </HStack>
+            </CardFooter>
+          </VStack>
+        </PopoverTrigger>
+        <PopoverContent mt={5} width={"348px"}>
+          <PopoverArrow />
+          <PopoverCloseButton />
```

Comment:
as per @AdamFinkle, the close button seems misaligned (also present in the design); can we check with @mjjsf if it's intended that it's different distances from top and right edge and, if so, correct in a follow-up PR?